sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_options.hpp

```diff
@@ -1144,6 +1149,11 @@ namespace Azure { namespace Storage { namespace Blobs {
      */
     Azure::Core::Context Context;
 
+    /**
+     * @brief Whether the existing blob should be deleted and recreated.
```

Comment:
Why block blob's comment is different with page blob/append blob's? I think block blob is also deleted and recreated if overwritten.

Reply:
align with .Net